kubernetes/plugin.go

```diff
@@ -146,7 +156,21 @@ func listK8sDynamicCRDs(ctx context.Context, cn *connection.ConnectionCache, c *
 			pageLeft = false
 		}
 
-		crds = append(crds, response.Items...)
+		for _, pattern := range filterCrds {
+			for _, item := range response.Items {
+				if ok, _ := path.Match(pattern, item.Name); ok {
+					if !helpers.StringSliceContains(temp_crds, item.Name) {
```

Comment:
What is this protecting against, the plugin attempting to create multiple tables with the same name? How does the plugin handle this today if it finds multiples matches?

Reply:
@cbruno10 in real time scenario, there should not be any name collision. Still, if a name collision happens, we'll handle it as discussed [here](https://turbothq.slack.com/archives/C01AC8JQNHH/p1677095863918979).

The above check restricts the plugin from creating multiple tables with the same name if the user provides it in the `custom_resource_tables` parameter.
Like, custom_resource_tables = ["volumesnapshotcontent", "volumesnapshotcontents.snapshot.storage.k8s.io"]
Here plugin should create only one table, `kubernetes_volumesnapshotcontent`.